<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="outlb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>60</red>
          <green>60</green>
          <blue>60</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>60</red>
          <green>60</green>
          <blue>60</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stylelb">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>60</red>
          <green>60</green>
          <blue>60</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>60</red>
          <green>60</green>
          <blue>60</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>76</red>
          <green>76</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="contentlb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="contentbt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>610</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>load content</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stylebt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>load style</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="iterationsSB">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSB">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>250</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthSB">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>250</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="style_weightSB">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>90</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="content_weightSB">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>130</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="prefix">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
    <property name="value">
     <double>0.020000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="wideo_weightSB">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>727</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wysokosc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Szerokosc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteracje</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>waga stylu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>waga zawartosci</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>waga wideo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wygenerowane:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="style_layers_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::IconMode</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>block1_conv1</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block1_conv2</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block1_pool</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block2_conv1</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block2_conv2</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bock2_pool</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_conv1</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_conv2</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_conv3</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_pool</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_conv1</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_conv2</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_conv3</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_pool</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_conv1</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_conv2</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_conv3</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_pool</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Warstwa zawartosci:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Warstwy stylu:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="content_layer_list">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>121</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::IconMode</enum>
    </property>
    <item>
     <property name="text">
      <string>block1_conv1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block1_conv2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block1_pool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block2_conv1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block2_conv2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bock2_pool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_conv1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_conv2</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_conv3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block3_pool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_conv1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_conv2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_conv3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block4_pool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_conv1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_conv2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_conv3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>block5_pool</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
